docs/topics/cli-run.adoc

```diff
@@ -76,7 +76,7 @@ To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler`
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/bin/windup-cli -Dwindup.decompiler=procyon \
+$ <{ProductShortName}_HOME>/bin/{mta-cli} -Dwindup.decompiler=procyon \
```

Comment:
The `-D` argument no longer exists

Reply:
I removed the entire section on overriding the decompiler , fixed in  https://github.com/windup/windup-documentation/pull/828